### PR TITLE
Dev to main

### PR DESCRIPTION
**Pull request by dataswifty**
## Description

Mandatory, provide a detailed description of the changes.

## Why

Optional i.e. "Why I have chosen package X, and not alternative package Y, Z"

## AC

Copy and paste from task

## Source

Link to story/task/bug

## Type

- [ ] Bug Fix
- [ ] Feature Addition
- [ ] Refactor
- [ ] HotFix

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: hat/conf/play.conf [hunk 1]
```diff
@@ -82,8 +82,18 @@ play {
...
+        "org.hatdex.hat.utils.LoggingFilter",
+        "play.filters.hosts.AllowedHostsFilter",
+        "play.filters.cors.CORSFilter",
+        "play.filters.gzip.GzipFilter",
+        "org.hatdex.hat.utils.TLSFilter"
+    ]
+
     //    enabled += "play.filters.headers.SecurityHeadersFilter"
+
     enabled += "play.filters.hosts.AllowedHostsFilter"
```
Comment (xavji) [inline hat/conf/play.conf:97]: all these three filters (line 97-99) are applied twice... not very efficient

File: hat/conf/play.conf [hunk 2]
```diff
@@ -82,8 +82,18 @@ play {
       allowed = [".hubofallthings.net", ".hubat.net", ${?HAT_DOMAIN}]
     }
 
-    enabled = []
+    enabled = [
+        "com.github.stijndehaes.playprometheusfilters.filters.StatusCounterFilter",
+        "com.github.stijndehaes.playprometheusfilters.filters.StatusAndRouteLatencyFilter",
```
Comment (xavji) [inline hat/conf/play.conf:87]: there is a single filter that can do the job of thoese two

Comment (dataswifty) [reply hat/conf/play.conf:87]: Fixed.

File: hat/conf/play.conf (hunk 1 above)
Comment (dataswifty) [reply hat/conf/play.conf:97]: Fixed

**Review by xavji — APPROVED** [review]

**Pull request merged**